README.md

```diff
@@ -1 +1,45 @@
-# ALMA-API-Example-usage
\ No newline at end of file
+# ALMA-API-Example-usage
+
+## Introduction
+The Library-system Alma from ExLibris offers some API-endpoint (see 
+[Developers Network](https://developers.exlibrisgroup.com/alma/apis/)). 
+Unit provides some applications and services that make use of those Alma-APIs.
+In order to communicate with those Alma-APIs you need to provide an ```API Key```.
+As long as Units applications and services are running on safespring-cloud, 
+we will use Vault to handle those ```API Key``` 
+(see vault-client project on [gitLab.bibsys.no](https://git.bibsys.no/infrastruktur/vault-client) for details).
```

Comment:
Er det noe vits å  vise intern lenker ?

Reply:
vault-client kunne kanskje vært flyttet til github, men bør vel sjekke med Thomas om det

Reply:
Jeg spør Thomas.

Reply:
Thomas mente at det ingenting i veien å flytte vault-client til github, men han ville ikke ofre tid for det.

Så har jeg fjernet lenken.